### PR TITLE
fix(htz-components): fix and clear console errors/warnings

### PR DESCRIPTION
**Pull request by TawKhoury**
affects: @haaretz/htz-components

## Clearing major errors and warnings appeared in console.

* #### Most changes has affected .md files and prop-type definitions. However, please review the code below and if anything you need has been changed let me know.

* #### This PR has NO intend of refactoring code. As the one remaining `console.warn` better have minor refactoring by producers (last update: The last console warn has been fixed in other branch).

* #### `StandardArticlePageLayout.md ` file has been deleted from htz-components (less duplicated components in styleguide) 

* #### Some may notice `parsing header X-XSS-Protection`  Error. It's a known bug in current Google Chrome and Chromium . (Update: The fix for this landed in Chrome 66.0.3336.0, which is still in Canary.  Stable soon). https://bugs.chromium.org/p/chromium/issues/detail?id=807304

 
Thanks and enjoy clean console ;)

### CODE REVIEW

File: packages/components/htz-components/src/components/CommentsSection/CommentSent.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const propTypes = {
   /**
    * A callback that gets the called when submitting the Form
    * @param {Boolean} - True if the user clicked the sign up button, false if clicked no thanks button
```
Comment (kedarguy) [inline packages/components/htz-components/src/components/CommentsSection/CommentSent.js]: @param {Boolean}  didSignUp

File: packages/components/htz-components/src/components/CommentsSection/CommentSent.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const propTypes = {
...
-   * @param {String} - The email the user entered
```
Comment (kedarguy) [inline packages/components/htz-components/src/components/CommentsSection/CommentSent.js]: @param {String} userEmail

File: packages/components/htz-components/src/components/CommentsSection/CommentsSection.js [hunk 3]
```diff
@@ -76,9 +76,9 @@ export class CommentsSection extends React.Component {
      * `onSubmitCaptcha` which executes the captcha gets called when the
      * reportAbuse event from `<CommentList />` gets called, it saves the commentId in
      * `<CommentSection/>` state to later use `onCaptchaResolve`
-     * @param {String} - commentId
+    //  * @param {String} - commentId
```
Comment (kedarguy) [inline packages/components/htz-components/src/components/CommentsSection/CommentsSection.js]: @param {String} commentId

File: packages/components/htz-components/src/components/CommentsSection/CommentsSection.js [hunk 4]
```diff
@@ -76,9 +76,9 @@ export class CommentsSection extends React.Component {
...
      * @param {String} - captchaKey
```
Comment (kedarguy) [inline packages/components/htz-components/src/components/CommentsSection/CommentsSection.js]: @param {String} captchaKey

File: packages/components/htz-components/src/components/CommentsSection/CommentSent.js (hunk 1 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/CommentsSection/CommentSent.js]: Thanks

File: packages/components/htz-components/src/components/CommentsSection/CommentSent.js (hunk 2 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/CommentsSection/CommentSent.js]: Thanks

File: packages/components/htz-components/src/components/CommentsSection/CommentsSection.js (hunk 3 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/CommentsSection/CommentsSection.js]: Thanks

File: packages/components/htz-components/src/components/CommentsSection/CommentsSection.js (hunk 4 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/CommentsSection/CommentsSection.js]: Done

File: packages/components/htz-components/src/components/ArticleImage/ArticleImage.js [hunk 5]
```diff
@@ -217,21 +215,22 @@ class ArticleImage extends React.Component {
...
-    const { viewMode, lastItem, title, credit, } = this.props;
+    const { viewMode, lastItem, title, credit, alt, } = this.props;
 
     return (
       <Fragment>
         <Wrapper viewMode={viewMode} lastItem={lastItem}>
           <ZoomWrapper onClick={this.toggleFullScreen} />
           <ImageElement {...this.props} />
-          <Caption caption={title} credit={credit} />
+          <Caption caption={title} credit={credit} alt={alt} />
```
Comment (EranM6) [inline packages/components/htz-components/src/components/ArticleImage/ArticleImage.js]: alt doesn't needed in caption

Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleImage/ArticleImage.js]: Done!

File: packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js [hunk 6]
```diff
@@ -40,11 +40,11 @@ const articleHeaderPropTypes = {
       hasEmailAlerts: PropTypes.bool,
       hasPushAlerts: PropTypes.bool,
       // eslint-disable-next-line react/forbid-prop-types
-      image: PropTypes.object,
+      image: PropTypes.string,
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js]: Why did you change `image` to be a string. It will be an object from PAPI

File: packages/components/htz-components/src/components/ArticleImage/ArticleImage.js [hunk 7]
```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types';
 import { createComponent, withTheme, } from 'react-fela';
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/ArticleImage/ArticleImage.js]: Please revert all changes in this file. It is being worked on in a branch and will change completely

File: packages/components/htz-components/src/components/ArticleImage/ArticleImage.md [hunk 8]
```diff
@@ -1,17 +1,20 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/ArticleImage/ArticleImage.md]: Please revert all changes in this file. It is being worked on in a branch and will change completely

File: packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js (hunk 6 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js]: Temporary fix, this component will be refactored soon. for now this component receive the image as string its PropTypes sets to object.

File: packages/components/htz-components/src/components/ArticleBody/ArticleBody.md [hunk 9]
```diff
@@ -32,6 +32,7 @@
       },
```
[the remark this answers is not in the document]
Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleBody/ArticleBody.md]: Alright

File: packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js (hunk 6 above)
Comment (TxHawks) [reply packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js]: the `object` prop-type represent the correct form. No need to insert a bug for a temporary fix.

File: packages/components/htz-components/src/components/ArticleImage/ArticleImage.md (hunk 8 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleImage/ArticleImage.md]: Done

File: packages/components/htz-components/src/components/ArticleImage/ArticleImage.js (hunk 7 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleImage/ArticleImage.js]: Done

File: packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js (hunk 6 above)
Comment (TawKhoury) [reply packages/components/htz-components/src/components/ArticleHeader/ArticleHeader.js]: Done
